imod/protocols/protocol_applyTransformationMatrix.py

```diff
@@ -97,6 +97,7 @@ def computeAlignmentStep(self, tsObjId):
                         "-output %(output)s " \
                         "-xform %(xform)s " \
                         "-bin %(bin)d " \
+                        "-antialias -1 " \
```

Comment:
I have a local change for this with a parameter, so user can choose the method.

Reply:
Let's merge this first and then my branch.

Reply:
I'm still fixing things though...